Source/WebCore/css/CSSStyleRule.cpp

```diff
@@ -313,4 +313,10 @@ CSSRuleList& CSSStyleRule::cssRules() const
     return *m_ruleListCSSOMWrapper;
 }
 
+void CSSStyleRule::getChildStyleSheets(HashSet<RefPtr<CSSStyleSheet>>& childStyleSheets)
+{
+    for (unsigned index = 0 ; index < nestedRules().size() ; index++)
```

Comment:
`nestedRules().size() -> length()`
`++index`

Reply:
Literally replace `nestedRules().size()` with `length()`. Not sure how I can be clearer.

Reply:
oh ok, updated both places